samples/meetings-attendance-report/csharp/README.md

```diff
@@ -0,0 +1,157 @@
+    -   Client secret for the bot which will be later used during Azure bot creation
+    -   Directory (tenant) ID
+We recommend that you copy these values into a text file, using an application like Notepad. We will need these values later.
+
+-  Under left menu, navigate to  **API Permissions**, and make sure to add the following permissions of Microsoft Graph API > Application permissions:
+    -  OnlineMeetingArtifact.Read.All
+
+Click on Add Permissions to commit your changes.
+
+- If you are logged in as the Global Administrator, click on the Grant admin consent for %tenant-name% button to grant admin consent else, inform your admin to do the same through the portal or follow the steps provided here to create a link and send it to your admin for consent.
+
+- Global Administrator can grant consent using following link:  [https://login.microsoftonline.com/common/adminconsent?client_id=](https://login.microsoftonline.com/common/adminconsent?client_id=)<%appId%> 
+
+### 3. Allow applications to access online meetings on behalf of a user
+- Follow this link- [Configure application access policy](https://docs.microsoft.com/en-us/graph/cloud-communication-online-meeting-application-access-policy)
+- **Note**: Copy the User Id you used to granting the policy. You need while configuring the appsettings.json file.
+
+### 4. Setup a Azure bot resource
+- Create new Azure Bot resource in Azure.
+- Select Type of App as "Multi Tenant"
+-  Select Creation type as "Use existing app registration"
+- Use the copied App Id and Client secret from above step and fill in App Id and App secret respectively.
+- Click on Create on the Azure bot.   
+- Go to the created resource, navigate to channels and add "Microsoft Teams".
+- Ensure that you've [enabled the Teams Channel](https://docs.microsoft.com/en-us/azure/bot-service/channel-connect-teams?view=azure-bot-service-4.0)
+
+
+### 5. Update the appsettings.json
+-  Update the `appsettings.json` configuration for the bot to use the `MicrosoftAppId` and `MicrosoftAppPassword` and `MicrosoftAppTenantId` and `AppBaseUrl` and `UserId` (Note that the MicrosoftAppId is the AppId created in step 2 , the MicrosoftAppPassword is referred to as the "client secret" in step 2 and you can always create a new client secret anytime., MicrosoftAppTenantId is reffered to as Directory tenant Id in step 2, AppBaseUrl is the URL that you get in step 1 after running ngrok, UserId of the user used while granting the policy in step 3). 
+
+
+### 6. Manually update the manifest.json
+- Edit the `manifest.json` contained in the  `/AppManifest` folder to and fill in App-Id this can be any GUID or your MicrosoftAppId and you need to fill MicrosoftAppId (that was created in step 2 and it is the same value of MicrosoftAppId as in `appsettings.json` file) *everywhere* you see the place holder string `<<Microsoft-App-Id>>` (depending on the scenario it may occur multiple times in the `manifest.json`) and
```

Comment:
<<Microsoft-App-Id>> => Verify the manifest GUID name and update it accordingly in Readme file.